apps/openmw/mwmechanics/character.cpp

```diff
@@ -983,7 +983,6 @@ void CharacterController::handleTextKey(const std::string &groupname, const std:
     if(evt.compare(0, 7, "sound: ") == 0)
     {
         MWBase::SoundManager *sndMgr = MWBase::Environment::get().getSoundManager();
-        sndMgr->stopSound3D(mPtr, evt.substr(7));
```

Comment:
These stopSound3D calls were added for reason: e71aa53574f29860f7bf1e4baa254a44cde78add
Make sure that you do not regress [bug #2689](https://gitlab.com/OpenMW/openmw/issues/2689).

Reply:
playSound3D already stops the sound.

Reply:
Yes it seems the stop was added a bit later (for a different bug) https://github.com/OpenMW/openmw/commit/1aaa8a76c5d9654669bf5c7ff0a408df5c3ef5a0